lib/internal/validators.js

```diff
@@ -130,6 +130,11 @@ function validateNumber(value, name) {
     throw new ERR_INVALID_ARG_TYPE(name, 'number', value);
 }
 
+function validateArray(value, name) {
+  if (!Array.isArray(value))
+    throw new ERR_INVALID_ARG_TYPE(name, 'array', value);
```

Comment:
Using the capitalized version seems to be what we do elsewhere for JS class names, I think? There are 6 places in `lib/` where we use `ERR_INVALID_ARG_TYPE` together with `'Array'`, as far as I can tell